### PR TITLE
Adding puppeteer-vanilla-example-project

### PR DESCRIPTION
**Pull request by ankit-dq**

### CODE REVIEW

File: Node/puppeteer_vanilla/package.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
+  "name": "pupeteer_vanilla",
+  "version": "1.0.0",
```
Comment (jthickens) [inline Node/puppeteer_vanilla/package.json:3]:
```suggestion
  "version": "4.4.0",
```

File: Node/puppeteer_vanilla/package.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
+{
+  "name": "pupeteer_vanilla",
```
Comment (jthickens) [inline Node/puppeteer_vanilla/package.json:2]:
```suggestion
  "name": "pupeteer",
```

File: Node/puppeteer_vanilla/tests/example.spec.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+        headless: false,
+    });
+    const page = await browser.newPage();
+    await page.goto("https://broken-workshop.dequelabs.com/");
+    // Uncomment the below line if screenshot is required
+    // await page.screenshot({
+    //     path: 'example_'  + '.png'
+    // });
+    const results = await new AxeDevToolsPuppeteer(page).analyze();
+    console.log(results);
```
Comment (jthickens) [inline Node/puppeteer_vanilla/tests/example.spec.js:18]: why are we console logging results here?

File: Node/puppeteer_vanilla/README.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+
+```sh
+npm install
+```
+
+## Run Tests
+
+The **_tests_** directory contains the example test file **`example.spec.js`** that analyzes the `https://broken-workshop.dequelabs.com/` for accessibility issues.
+
+The following command runs the test files in the **_tests_** directory.
```
Comment (jthickens) [inline Node/puppeteer_vanilla/README.md:39]:
```suggestion
The following command runs the test file in the **_tests_** directory.
```

File: Node/puppeteer_vanilla/README.md [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+
+**`npm test`** runs the **`rimraf`** command mentioned in the **`package.json`**.
+
+```json
+  "scripts": {
+    "test": "rimraf ./a11y-results && node ./tests/*.spec.js"
+  },
+```
+
+Every time you run the **`npm test`**, the **`rimraf`** command clears all saved results from the **`a11y-results`** directory, so if you want to retain previous test results, you should remove **`rimraf`** from the **`scripts`** object.
```
Comment (jthickens) [inline Node/puppeteer_vanilla/README.md:61]:
```suggestion
Every time you run **`npm test`**, the **`rimraf`** command clears all saved results from the **`a11y-results`** directory, so if you want to retain previous test results, you should remove **`rimraf`** from the **`scripts`** object. 
```

File: Node/puppeteer_vanilla/README.md [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+
+```json
+  "scripts": {
+    "test": "rimraf ./a11y-results && node ./tests/*.spec.js"
+  },
+```
+
+Every time you run the **`npm test`**, the **`rimraf`** command clears all saved results from the **`a11y-results`** directory, so if you want to retain previous test results, you should remove **`rimraf`** from the **`scripts`** object. 
+
+If you want to modify this project and publish your results in a different folder other than **`./ally-results`**, you should update the output directory in the test file and the **test** parameter.
```
Comment (jthickens) [inline Node/puppeteer_vanilla/README.md:63]:
```suggestion
If you want to modify this project and publish your results in a folder other than **`./ally-results`**, you should update the output directory in the test file and the **test** parameter. 
```

**Review by jthickens — DISMISSED** [review]
Also, .DS_Store files need to be removed

File: Node/puppeteer_vanilla/tests/example.spec.js (hunk 3 above)
Comment (ankit-dq) [reply Node/puppeteer_vanilla/tests/example.spec.js]: removed the console logging. Was just for the purpose of the user to see the results in the logs if needed by them.

File: Node/puppeteer/README.md [hunk 7]
```diff
@@ -0,0 +1,71 @@
...
+
+## Clone Project
+
+Follow these steps to clone and navigate to the directory:
+1. Clone this repo from GitHub.
+2. Open the project in your favourite editor.
+3. Navigate from the root of the repo to this example with the following command:
+
+```sh
+cd Node/puppeteer_vanilla
```
Comment (jthickens) [inline Node/puppeteer/README.md:19]:
```suggestion
cd Node/puppeteer
```
```suggestion
cd Node/puppeteer_vanilla
```

File: Node/puppeteer/README.md [hunk 8]
```diff
@@ -0,0 +1,71 @@
...
+```
+
+## Install Dependencies
+
+Install the dependencies including **axe DevTools Puppeteer** and **axe DevTools Reporter** for the project.
```
Comment (jthickens) [inline Node/puppeteer/README.md:24]:
```suggestion
Install the dependencies including **axe DevTools Puppeteer** and **DevTools Reporter** for the project.
```
```suggestion
Install the dependencies including **axe DevTools Puppeteer** and **axe DevTools Reporter** for the project.
```

Comment (jthickens) [inline Node/puppeteer/README.md:24]:
```suggestion
Install the dependencies including **axe DevTools Puppeteer** and **DevTools Reporter** for the project.
```
```suggestion
Install the dependencies including **axe DevTools Puppeteer** and **axe DevTools Reporter** for the project.
```

File: Node/puppeteer/README.md [hunk 9]
```diff
@@ -0,0 +1,71 @@
...
+
+The following command installs all the required dependencies to run this example project.
+
+```sh
+npm install
+```
+
+## Run Tests
+
+The **_tests_** directory contains the example test file **`example.spec.js`** that analyzes the `https://broken-workshop.dequelabs.com/` for accessibility issues.
```
Comment (jthickens) [inline Node/puppeteer/README.md:37]:
```suggestion
The **_tests_** directory contains the example test file **`example.spec.js`** that analyzes `https://broken-workshop.dequelabs.com/` for accessibility issues.
```
```suggestion
The **_tests_** directory contains the example test file **`example.spec.js`** that analyzes the `https://broken-workshop.dequelabs.com/` for accessibility issues.
```

Comment (jthickens) [inline Node/puppeteer/README.md:37]:
```suggestion
The **_tests_** directory contains the example test file **`example.spec.js`** that analyzes `https://broken-workshop.dequelabs.com/` for accessibility issues.
```
```suggestion
The **_tests_** directory contains the example test file **`example.spec.js`** that analyzes the `https://broken-workshop.dequelabs.com/` for accessibility issues.
```

File: Node/puppeteer/tests/example.spec.js [hunk 10]
```diff
@@ -0,0 +1,31 @@
...
+const reporter = new Reporter('Basic Test Suite', './a11y-results/json');
+var assert = require('assert');
+
+(async () => {
+    const browser = await puppeteer.launch({
+        headless: false,
+    });
+    const page = await browser.newPage();
+    await page.goto("https://broken-workshop.dequelabs.com/");
+    // Uncomment the below line if screenshot is required
```
Comment (jthickens) [inline Node/puppeteer/tests/example.spec.js:13]: screenshot code should be removed

Comment (jthickens) [inline Node/puppeteer/tests/example.spec.js:13]: screenshot code should be removed

**Review by jthickens — CHANGES_REQUESTED** [review]

File: Node/puppeteer/README.md (hunk 8 above)
Comment (jthickens) [reply Node/puppeteer/README.md:24]: same as in playwright project. This is an intentional naming scheme

File: Node/puppeteer/README.md [hunk 11]
```diff
@@ -0,0 +1,71 @@
...
```
[the remark this answers is not in the document]
Comment (jthickens) [reply Node/puppeteer/README.md:37]: "the" alone isn't correct. "the page" or removing "the" is acceptable.

File: Node/puppeteer/README.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (nagaraju-vadakoppula) [reply Node/puppeteer/README.md]: as per the directory, **puppeteer** was given instead of **puppeteer_vanilla**

**Review by jthickens — APPROVED** [review]

File: Node/puppeteer/tests/example.spec.js [hunk 12]
```diff
@@ -0,0 +1,27 @@
+const puppeteer = require('puppeteer');
+const { AxeDevToolsPuppeteer } = require('@axe-devtools/puppeteer');
+const Reporter = require('@axe-devtools/reporter').default;
+const reporter = new Reporter('Basic Test Suite', './a11y-results/json');
+var assert = require('assert');
```
Comment (michael-siek) [inline Node/puppeteer/tests/example.spec.js:5]:
```suggestion
const assert = require('assert');
```

**Review by michael-siek — DISMISSED** [review]

**Pull request merged**